src/computercraft/installasjon/installasjon.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Installering av ComputerCraft
+level: 1
+author: Geir Arne Hjelle
+indexed: False
+---
+
+# Introduksjon {.intro}
+
+[ComputerCraft](http://www.computercraft.info/) er en mod til
+[Minecraft](https://minecraft.net/), som gir deg muligheten til å
+bygge og programmere **datamaskiner** og **roboter** inne i
+Minecraft-verdenen.
+
+![](../introduksjon_til_computercraft/introduksjon_til_computercraft.png)
+
+# Installering av ComputerCraft {.activity}
+
+ComputerCraft er en mod til Minecraft, og krever derfor at Minecraft
+allerede er installert på datamaskinen. Vi antar at du er kjent med og
+har installert Minecraft.
+
+ComputerCraft er typisk ikke tilgjengelig for siste versjon av
+Minecraft. Legg derfor merke til hvilken versjon av Minecraft som
+kreves (i eksemplene nedenfor kreves versjon 1.7.10, men dette kan
+endre seg), og pass på at du installerer riktig versjon av Forge.
+
+## Sjekkliste {.check}
+
++ Gå til
+  [computercraft.info/download](http://www.computercraft.info/download/).
+  Bla nedover på siden til du finner et avsnitt som heter **Download
+  and installing**. Last ned ComputerCraft til datamaskinen din, og
+  noter deg hvilken versjon av Minecraft denne er kompatibel med (for
+  eksempel versjon 1.7.10). _Ikke_ prøv å kjør filen, vi skal kopiere
+  den til riktig sted litt senere.
+
+    [![](last_ned_computercraft.png)](http://www.computercraft.info/download/)
+
++ Start **Minecraft Launcher**. Sjekk om du har den riktige versjonen
+  av Minecraft tilgjengelig. Hvis ikke kan du velge `New Profile` og i
+  nedtrekksmenyen `Use Version` velger du riktig versjon. Gi profilen
+  et navn (for eksempel versjonsnummeret) og klikk `Save
+  Profile`. Klikk `Play` slik at denne versjonen blir lastet ned og
+  startet opp. Avslutt spillet.
+
+    [![](minecraft_versjon.png)]()
```

Comment:
Det er pga [denne](https://github.com/arve0/codeclub_lesson_builder/commit/9172582475b723834d36419bdf7cb2155187211c), alle bilder som er på egen linje blir midtjustert.

Mener bildene var sentrert etter kommentar fra @hallvard (?), som mente det var ryddigere.

To løsninger:
- La bildet være i teksten: `tekst bla bla. ![](bilde.png)`
  - Ulemper: Det blir lite rom rundt bildet. Om man legger til rom, vil dette gjelde alle bilder, også bilder som ikke gir mening å ha rom rundt (eksempelvis knapper som er lagt inline i teksten).
- Fjerne sentreringen fra `figure > img`.